webpack/stories/docs/intervalMiddleware.md

```diff
@@ -69,4 +81,13 @@ handlePolling = () => {
   // use the same key you used to start the interval.
   dispatch(stopInterval(key))
 }
+```
+
+or you can generate your own action with the same type as STOP_INTERVAL,
+don't forget to pass the unique 'key' in the payload, and we will catch it already - though this is less recommended.
+```js
+export const stopInterval = key => ({
+  type: STOP_INTERVAL,
+  key,
+});
```

Comment:
This is just re-creating the `stopInterval` action creator.  I don't see a reason to recommend this.

Reply:
agreed